src/dialogs/more-info/controls/more-info-weather.ts

```diff
@@ -267,6 +267,7 @@ class MoreInfoWeather extends LitElement {
     return date.toLocaleDateString(this.hass.language, {
```

Comment:
We should use <https://github.com/home-assistant/frontend/blob/dev/src/common/datetime/format_date.ts#L4>

Reply:
https://github.com/home-assistant/frontend/blob/dev/src/common/datetime/format_time.ts#L4